crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -600,7 +664,15 @@ where
         } else {
             match &v.raw {
                 Some(raw) => {
-                    if self.cfg.target < EsVersion::Es2021 && raw.contains('_') {
+                    if raw.len() > 2 && self.cfg.target < EsVersion::Es2015 && {
```

Comment:
I don't believe any engine supports bigint but not Es2015.
Can we ignore this compatibility? @kdy1

Reply:
Yeah I think so